### ISSUE TITLE
2019-11-27 tc018 M4SS comp engine error when parameterset set to "PARAMETERDISPLAYLIST"

### ISSUE THREAD

**Issue by jhhughes256** [tensfeldt/openNCA#63]
### Error

When running tc018 with `parameterset <- "PARAMETERDISPLAYLIST"` the computation error shown below occurs. 

```
Error in value[[3L]](cond) : 
  Error in `[<-.data.frame`(`*tmp*`, i, , value = c(43456658, 527333.333333333, : replacement has 16 items, need 17
For SDEID 43456658 
7. stop(paste0(e, "For SDEID ", unique(data_data[, map_data$SDEID])[i])) at run_M4_SS_computation.R#1409
6. value[[3L]](cond) 
5. tryCatchOne(expr, names, parentenv, handlers[[1L]]) 
4. tryCatchList(expr, classes, parentenv, handlers) 
3. tryCatch({
    if (comp_required[["DIi"]]) {
        di <- list()
    } ... at run_M4_SS_computation.R#676
2. run_M4_SS_computation(data = merged_data, map = map_data, method = method, 
    model_regex = model_regex, parameter_list = parameter_list, 
    return_list = return_list) at run_computation.R#2320
1. run_computation(data = d, map = mct, flag = flags, parameterset = parameterset) 
```

The variables asked for by PARAMETERDISPLAYLIST are:

```
> mct$PARAMETERDISPLAYLIST
[1] "AUCINFP;AUCLASTi;AUCLASTiDN;AUCLASTCi;AUCTAUi;AUCXPCTP;AURCINFP;AURCLAST;AURCXPCTP;CMAXiDN;KEL;KELRSQ;KELNOPT;MAXRATEi;THALF;THALFF;TLAST;TMAXi;TMAXRATEi"
```

While some of the variables seemed strange to me (i.e. asking for AUC instead of AURC, having i before DN for some variables) when changing these the error still occurs.

Note when appending the unchanged values for PARAMETERDISPLAYLIST to PARAMETERLIST **the error does not occur**. This seems to imply there is no issues with the values for `mct$PARAMETERDISPLAYLIST` but instead issues with what values are not in there. An example of the resulting parameterset is shown below.

```
> mct$PARAMETERDISPLAYLIST
[1] "AE;AEPCT;AET;AETAUi;AETAUpti;AETPCT;AT;AURCALL;AURCINFO;AURCINFP;AURCLAST;AURCT;AURCT1_T2;AURCXPCTO;AURCXPCTP;CLRT;CLRTAUi;DOSEC;DIi;DOSEi;FA;KEL;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;MAXRATE;MAXRATEi;MIDPTA;MIDPTLASTi;MIDPTN;RATEA;RATELASTi;RATEN;TAUi;THALF;THALFF;TLAG;TMAXRATE;TMAXRATEi;TOLDi;VOLSUM;FLGACCEPTKEL;FLGACCEPTPREDOSE;FLGACCEPTTAU;FLGACCEPTTMAX;AUCINFP;AUCLASTi;AUCLASTiDN;AUCLASTCi;AUCTAUi;AUCXPCTP;AURCINFP;AURCLAST;AURCXPCTP;CMAXiDN;KEL;KELRSQ;KELNOPT;MAXRATEi;THALF;THALFF;TLAST;TMAXi;TMAXRATEi"
```

**Comment by rudraya** [comment]
This issue was addressed in 6e99fab, please let me know if it still occurs.

**Comment by tensfeldt** [comment]
This was tested successfully with commit b8deb76 and can be closed.

**Comment by jhhughes256** [comment]
### Reopen

As of commit bb8e82dbcc I can replicate this issue again. Reopening.

**Comment by rudraya** [comment]
This issue was addressed in fce7a3c, please let me know if it still occurs.

**Comment by jhhughes256** [comment]
### Update

This issue no longer occurs as of  fce7a3c. Closing for now.
